### PR TITLE
Test all filters against random data to detect exceptions

### PR DESCRIPTION
**Pull request by shopmike**
Built this test out of curiosity. The good news is it passes. 💯 

@Shopify/liquid @Shopify/guardians-of-the-liquid

### CODE REVIEW

File: test/integration/standard_filter_test.rb [hunk 1]
```diff
@@ -754,6 +754,53 @@ def test_where_array_of_only_unindexable_values
...
+            @filters.send(method, *inputs)
+          rescue Liquid::ArgumentError => e
+            errors << "#{method} returned #{e.message}"
+          rescue Liquid::ZeroDivisionError => e
+            errors << "#{method} returned #{e.message}"
+          end
+        end
+      end
+    end
+    errors
```
Comment (Thibaut) [inline test/integration/standard_filter_test.rb]: This doesn't seem to be used for anything.

Comment (shopmike) [reply test/integration/standard_filter_test.rb]: I can pull these out, however, I might just use this to assert that it doesn't contain an incorrect parameter count error in any response.

Comment (shopmike) [reply test/integration/standard_filter_test.rb]: test my test

Comment (shopmike) [reply test/integration/standard_filter_test.rb]: Not needed as realized this is outside of Strainer so the exception is still thrown.

**Pull request merged**